int64 x 
bool person_present